tests/Directory.Build.props

```diff
@@ -14,14 +14,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.core" Version="2.3.0-beta2-build3683" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0-beta2-build1317" />
-    <PackageReference Include="Microsoft.TestPlatform.TestHost" Version="15.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
+    <PackageReference Include="xunit.core" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+    <PackageReference Include="XunitXml.TestLogger" Version="2.0.0" />

```

Comment:
There's no built-in XML logger. This is the one recommended on [this page](https://github.com/Microsoft/vstest-docs/blob/master/docs/report.md#available-test-loggers). However I see 2 issues with it:
- it's not an official implementation (https://github.com/spekt/xunit.testlogger)
- it doesn't take the framework into account, so there's only one result file, which seems to contain results only for the last framework run.

Alternative options:
- Do we really need those files? Me, I never looked at them. Maybe we could just disable reporting.
- Do we really need them to be in the xUnit format? We could use the built-in trx logger

Reply:
> it doesn't take the framework into account, so there's only one result file, which seems to contain results only for the last framework run.

The trx logger has the same problem. So if we want per-framework result files, we have to run the tests per framework, as @adamralph did in #1442...

Reply:
It's true. On the other hand, I'm not against submitting an issue/PR to spekt/xunit.testlogger.

Reply:
It's true. And the project is an absolute mess.

Reply:
I mostly enjoy the results files when the console output is too hard to read. Seems like we're making that easier, so I'm game to try without them.